### ISSUE TITLE
Info about chaff in Electronic warfare panel

### ISSUE THREAD

**Issue by GreyW3** [BattletechModders/LowVisibility#30]
Could you add digital info on chaffs to the electronic warfare panel? 
The chaff info  in the game is now extremely limited, in general it is unclear whether the effect of multiple hits is stacked, and what it is. 
chaffs
[https://roguetech.gamepedia.com/Weapons/Support/Chaff_CounterMeasures](url)
panel
[https://cdn.discordapp.com/attachments/686305981050781699/686307687868530766/BattleTech_2020_03_07_17_14_02_203_cr.png](url)

**Comment by boycad** [comment]
Information is a pervasive in Roguetech as a whole.  Despite oodles of tooltips and an apparently huge amount of effort going in to communicating with the player, it is too often incomprehensible.

I also dont understand how chaff works, depite trying.  But I also still dont know what "+1 ini Injury resist" means, for one example.  WTF is "ini"?  That isnt a word. Is it a contraction of "initial"?  Should there be a dot "ini."?

Anyways, why Im commenting here is because it seems like the Rogutech Wiki is really where all of this information problem should be addressed.  That is where stuff and systems should be written down and fully explained and illustrated, in complete sentences, 
with table and examples and references to other related subjects, like wikis are meant to be.  Because its too much for tooltips alone to handle, and answering your question here wont help many other people unless its then entered on the wiki.  

For instance, it isnt possible to explain how chaff works, without first explaining the  detection system it is a part of, is it?  

Basically, the mod authors should not be responsible for this aspect of the game; for explaining things that is.  The involvedc player should be.  Or thats how I see it.  

(They can also be very touchy about their mods not making sense.)

**Comment by IceRaptor** [comment]
It's not easy to integrate information about Chaff, because Chaff isn't something my mod provides directly. It's a combination of different effects (https://roguetech.fandom.com/wiki/Gear/Chaff_CounterMeasures). Those effects appear to be related to LowVis, and they sorta are, but largely aren't. For instance many things change visibility or sensor visibility, but these are vanilla concepts that I'm just building on the back of. 

Because of this, and the multitude of different effects that can potentially impact signature / visibility / etc I'm not willing to invest in a major improvement in this area. Sorry, but it's just not a good fit. Thanks for the input.
